### ISSUE TITLE
[MRESOURCES-102] Maven replace var project.build.directory for escape character '/'

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-resources-plugin#274]
**[Jose Morales Martinez](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=jomm)** opened **[MRESOURCES-102](https://issues.apache.org/jira/browse/MRESOURCES-102?redirect=false)** and commented

When I process Maven vars in resources files, Maven replace project.build.directory var  for path with characters //.

Maven 2.0.9 replace path successfully, but Maven 2.0.10 replace character '/' for '//'

---

**Affects:** 2.2

1 votes, 1 watchers

**Comment by jira-importer** [comment]
**[Brett Porter](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=brettporter)** commented

this is probably a difference between the resources plugin v2.2 and v2.3, not a bug in Maven.

Can you set the version of the resources plugin to 2.2 and confirm it works consistently in both versions?

If so, it might be a regression change that needs to be addressed there. A test project would be helpful as well.

**Comment by jira-importer** [comment]
**[Olivier Lamy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=olamy)** commented

Probably MSHARED-78 ?
Can you test with last resources plugin snapshot ?

**Comment by jira-importer** [comment]
**[Jose Morales Martinez](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=jomm)** commented

I use this configuration in pom.xml:

```
 
<plugin>
    <groupId>org.apache.maven.plugins</groupId>
    <artifactId>maven-resources-plugin</artifactId>
    <configuration>
        <encoding>UTF-8</encoding>
    </configuration>
</plugin>
```

I think this should use the last release plugin (2.3). I used this configuration with Maven 2.0.9 and go OK, but when I installed Maven 2.0.10, I have got this problem.

I am going swap resource plugin to version 2.2 and I will test with Maven 2.0.10 and comment the result.

**Comment by jira-importer** [comment]
**[Brett Porter](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=brettporter)** commented

how did that test go Jose?

**Comment by jira-importer** [comment]
**[Jose Morales Martinez](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=jomm)** commented

I fixed process-resources version to 2.2  and then it works successfully. With version 2.3 it doesn't work.

You can close this issue.
